hyperspyui/plugins/dpc_segmented_get_shifts.py

```diff
@@ -0,0 +1,33 @@
+from hyperspyui.plugins.plugin import Plugin
+import numpy as np
+import hyperspy.api as hs
+
+
+class DpcFftMask(Plugin):
+    name = "Beam shifts from segmented"
+
+    def create_actions(self):
+        self.add_action(self.name + '.default', self.name, self.default,
+                        tip="")
```

Comment:
You could change the name of the plugin to something more descriptive. Then this could be something like this:
```python
self.add_action(self.name + '.<action key here>', '<Menu entry text here>, self.default, tip="<tooltip for menu entry here>")
```

Then the line in `create_menu` becomes:
```python
self.add_menuitem('Magnetic', self.ui.actions[self.name + '.<action key here>'])
```